tc_generator/tc_gen.py

```diff
@@ -53,6 +53,12 @@
         'link': 'https://www.python.org/downloads/',
         'compile': ''
     },
+    {
+        'req': 'C#',
+        'command': 'logic',
```

Comment:
Is command just `logic`?

Reply:
I have made the necessary changes. Please do have a look.